### ISSUE TITLE
WCAG 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by kosarko** [ufal/dspace-angular#12]
https://www.w3.org/TR/WCAG21/#info-and-relationships

- Lists do not contain only `<li>` elements and script supporting elements (<script> and <template>): br-elements are in use im list (probably just footer)
-  the hiearchie of h-elements are not followed, h1 is missing

**Comment by ToneMereteBruvik** [comment]
Note that there are isses with the hierarchie  for instance on this page: https://lindat.mff.cuni.cz/repository-ng/items/00d07d78-a3ba-4e3c-8382-938cbeefe83c

![Image](https://github.com/user-attachments/assets/60ba81f2-94a8-4ba9-865a-2a7ccd7a31c8)

**Comment by ToneMereteBruvik** [comment]
I tested a bit using VoiceOver (a build in screen read on Apple OS). When a search is done there are no information given by the VoiceOver what number of hits you get. Something have to be done with it.
